server/src/MergeRequestAcceptor.ts

```diff
@@ -225,6 +225,13 @@ export const acceptMergeRequest = async (
 	console.log(`[MR][${mergeRequestIid}] Calling merge request`);
 	const mergeRequestInfo = await gitlabApi.getMergeRequestInfo(projectId, mergeRequestIid);
 
+	if (config.PROJECT_ID !== '' && config.PROJECT_ID !== mergeRequestInfo.project_id.toString()) {
```

Comment:
If I understand the motivation correctly, the bot should skip such foreign MRs completely. This change does not prevent such MRs to queue or to be rebased.

The check should be instead moved to https://github.com/shipmonk-rnd/gitlab-merger-bot/blob/e708d10b5cf568ff5a4102e762de9211f050f1f3/server/src/MergeRequestReceiver.ts#L16 or, even better https://github.com/shipmonk-rnd/gitlab-merger-bot/blob/e708d10b5cf568ff5a4102e762de9211f050f1f3/server/src/GitlabApi.ts#L178

Reply:
When I keep on this place is automatically unassigned.

Reply:
I don't see reason why the bot should do **anything** on projects that are not configured 🤔

Reply:
It should be unassigned. 
It prevents spam logs about "ignoring MR".

Reply:
I had to check it locally to see it properly which lead me to make some changes. Please take a look and preferrably add a test case.

I moved the check to `prepareMergeRequestForMerge` which now:
- prevents merge of the MR as it's not even put to the queue
- it unassigns the Assignee

Reply:
test added, it looks better. thx.